Sources/tart/Commands/Set.swift

```diff
@@ -51,6 +60,17 @@ struct Set: AsyncParsableCommand {
       }
     }
 
+    if randomMAC {
+      vmConfig.macAddress = VZMACAddress.randomLocallyAdministered()
+    }
+
+    #if arch(arm64)
+      if randomSerial {
+        let oldPlatform = vmConfig.platform as! Darwin
```

Comment:
Should we not assume that it's always Darwin and do a proper type check & throw an error if it's Linux?

Reply:
Though about it but was too lazy and the error message will be self-explanatory. Plus help mentions macOS.